lib/exercism/guest.rb

```diff
@@ -3,6 +3,7 @@
 class Guest
   def id
   end
+  alias_method :flipper_id, :id
```

Comment:
Given that the `flipper_id` for `User` is the `username`, it kinda makes sense to maintain it here too.

```
def flipper_id
  username
end
```

This is a nitpick, feel free to just ignore this. 😄